spec/owner_spec.rb

```diff
@@ -49,21 +49,30 @@
     expect(owner.pets).to eq({:fishes => [], :dogs => [], :cats => []})
   end
 
-  it 'can buy a fish' do 
+  it 'can buy a fish that is an instance of the Fish class' do 
```

Comment:
This is a lot of expects in one `it` block. Feels more like this wants to be moved into a `describe` block with a few different `it` blocks that test the different things this one `it` block is teating